scipy/linalg/_matfuncs_sqrtm.py

```diff
@@ -84,7 +84,10 @@ def _sqrtm_triu(T, blocksize=64):
             start += size
 
     # Within-block interactions (Cythonized)
-    within_block_loop(R, T, start_stop_pairs, nblocks)
+    try:
+        within_block_loop(R, T, start_stop_pairs, nblocks)
+    except RuntimeError as e:
+        raise SqrtmError(*e.args)
```

Comment:
I think it should be
```suggestion
        raise SqrtmError(*e.args) from e
```

Reply:
I'm so glad I lerant something today. Thanks @tupui, that syntax was new to me!